public_html/wp-content/themes/wporg-events-2023/style.css

```diff
@@ -1,18 +1,10 @@
 /*
-Theme Name: Events.WordPress.org 2023
-Theme URI: https://wordpress.org
-Author: the WordPress.org team
-Author URI: https://wordpress.org
-Description: Includes templates for the homepage, event archives, etc.
-Requires at least: 6.4
-Tested up to: 6.5
-Requires PHP: 7.0
-Version: 1.0
-License: GNU General Public License v2 or later
-License URI: http://www.gnu.org/licenses/gpl-2.0.html
-Text Domain: wordcamporg
-Template:    wporg-parent-2021
-*/
+ * Theme Name:  Events.WordPress.org 2023
```

Comment:
looks like maybe a code editor has changed this - can it be reverted to the current version from the `production` branch?

Reply:
Oh...
I'll revert this change.